### ISSUE TITLE
Consider removing query parameters "supportsSound" and "supportsEnhancedSound"

### ISSUE THREAD

**Issue by jbphet** [phetsims/tambo#50]
The `supportsSound` and `supportsEnhancedSound` query parameters were added a few months back as a way to support the development of sonification on the master branch for a simulation without enabling sound generation by default.  The idea was borrowed from how accessibility is currently being developed.  The primary motivation for this is to prevent the accidental deployment of a sim with an unfinished sonification implementation.

In a recent discussion, @samreid suggested that we get rid of these query parameters and control sound behavior entirely through options to the `Sim` constructor.

**Comment by jbphet** [comment]
This topic was discussed at the 9/27/2018 developer meeting, and at the end of that meeting we had decided that I should move forward with the removal of these query parameters with the caveat that I should discuss it with @emily-phet first (@kathy-phet suggested that).  @emily-phet and I discussed it today, and we decided that we should actually keep these around, at least for now.  Our reasoning is that, while the recent implementation of sonification for John Travoltage was reasonably quick, other sims in the future might take significantly longer because we may have to do prototype development of sound designs for sims that don't have phet-io implementations.  The reason that having a phet-io implementation matters is that our current approach for prototyping sound designs is to create a "sonification wrapper" that relies on phet-io to monitor the events and state within the sim.  If the sim is not phet-io instrumented, this option isn't available.  Sonification **could** be developed on a branch, but being able to do it on master without risking deployment seems to both of us like a better option.

So, bottom line, I'm leaving this for now, and I logged this issue to have a record of the thought process.

Tagging @samreid, @pixelzoom, and @ariel-phet so that they are aware of the current thinking and can chime in if they have strong objections.  If not, I'll close this in a couple of days.

**Comment by samreid** [comment]
The question of what's best seems to hinge on:

(1) How much longer do you mean by "significantly longer"?  6 months?  More?
(2) How often do we expect to publish a sim *from master* after sonification begins and before it is complete?

If (1) and (2) are both sufficiently high then the query parameters seem best.  I don't know either of those numbers, so I am not qualified to decide.

**Comment by jbphet** [comment]
It's hard to quantify precisely, but @emily-phet described a possible situation where we start on the sound design implementation for a sim, then priorities shift and the complete implementation is delayed.  So, I think several months is a possibility.

I'm going to leave the query params for now and close this issue.  We can always remove them later if we decide they really are unnecessary.
